### PR TITLE
Embed file permissions for features

### LINKED ISSUE #1895: elemental-setup-*.service files have executable bit set
And systemd complains loudly about it.

### PR DESCRIPTION
**Pull request by frelon**
This PR fixes the following warnings on boot:

```
Feb 16 14:33:07 localhost systemd[1]: Configuration file /usr/lib/systemd/system/elemental-setup-initramfs.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Feb 16 14:33:07 localhost systemd[1]: Configuration file /usr/lib/systemd/system/elemental-rootfs.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Feb 16 14:33:07 localhost systemd[1]: Configuration file /usr/lib/systemd/system/elemental-setup-rootfs.service is marked executable. Please remove executable permission bits. Proceeding anyway.
....
```

It does this by collecting the features into tarballs before embedding them into the go binary.

Each feature gets it's own tarball and is generated in `pkg/features/generated/<feature-name>.tar`. The generation happens in the `pkg/features/generate-tarballs.go` generator which makes it necessary to run `go generate ./...` before building.

Fixes #1895

### CODE REVIEW

File: pkg/features/features_test.go [hunk 1]
```diff
@@ -41,5 +41,8 @@ var _ = Describe("Init Action", func() {
 			_, err := features.Get([]string{"unknown-abc"})
 			Expect(err).To(HaveOccurred())
 		})
+		It("Installs a feature correctly", func() {
+
```
Comment (anmazzotti) [inline pkg/features/features_test.go:45]: Test missing here?

**Review by anmazzotti — COMMENTED** [review]
Rest looks good! Nice!

**Review by davidcassany — APPROVED** [review]
Nice! :+1:

File: pkg/features/features_test.go (hunk 1 above)
Comment (frelon) [reply pkg/features/features_test.go:45]: Hmm, this code is covered by `pkg/action/init_test.go`, removing this!

**Pull request merged**